asv_bench/benchmarks/omnisci/benchmarks.py

```diff
@@ -244,25 +244,43 @@ def create_fillna_value(value_type: str, columns: list):
         return value
 
 
-class TimeValueCountsSeries:
-    param_names = ["shape", "ngroups"]
-    params = [
-        get_benchmark_shapes("omnisci.TimeValueCountsSeries"),
-        GROUPBY_NGROUPS,
-    ]
-
-    def setup(self, shape, ngroups):
+class BaseTimeValueCounts:
+    def setup(self, shape, ngroups=5, subset=1):
         ngroups = translator_groupby_ngroups(ngroups, shape)
-        self.df, self.column_names = generate_dataframe(
+        self.df, self.subset = generate_dataframe(
             ASV_USE_IMPL,
             "int",
             *shape,
             RAND_LOW,
             RAND_HIGH,
-            groupby_ncols=1,
+            groupby_ncols=subset,
             count_groups=ngroups,
         )
-        self.series = self.df[self.column_names[0]]
+        trigger_import(self.df)
+
+
+class TimeValueCountsDataFrame(BaseTimeValueCounts):
```

Comment:
The following change should be introduced as part of a separate issue for pandas on ray benchmarks. `TimeValueCountsFrame` -> `TimeValueCountsDataFrame`